recipe/meta.yaml

```diff
@@ -22,13 +22,13 @@ requirements:
   host:
     - python
     - pip
-    - openssl
+    - openssl  # [not linux]
     - postgresql
   run:
     - python
     # libpq is a repackaging of only the library to connect to postgres
     - libpq
-    - openssl
+    - openssl  # [not linux]
```

Comment:
I'm not an expert, can you explain a bit why this is needed? Actually, not needed?

Reply:
The `setup.cfg` for this package only configures links against libcrypto for Darwin and Windows:

https://github.com/psycopg/psycopg2/blob/2_8_6/setup.py#L369-L379

So, the openssl requirement for linux is just cruft. You can see the overdepending warning [here](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=255827&view=logs&j=bb1c2637-64c6-57bd-9ea6-93823b2df951&t=350df31b-3291-5209-0bb7-031395f0baa1&l=734) for previous builds.

[I'm a pedant and always run builds with `conda build --error-overlinking --error-overdepending`, so without this change, the build fails for me on linux]